src/app/src/components/annotations/annotator.tsx

```diff
@@ -901,6 +932,10 @@ export default class Annotator extends Component<
           }, 150);
           /* Reset to Default Zoom */
           this.map.setMinZoom(-3);
+          /** Get inference from cache */
+          if (asset.isCached) {
+            this.getInference(false);
```

Comment:
mentioned above

Reply:
this is the only time you need to keep this as false

Reply:
```javascript
/* Check if Image is Cached */
this.getInference(asset.isCached === false)
```

Can be considered, but this is kind of up to your personal taste on how descriptive the sentence is

Reply:
Using back 
```
/** Get inference if Image is Cached */
if (asset.isCached)  this.getInference(false);
```
so that the inference api wont be called when selecting images that are not cached